### PR TITLE
metrics: Update fast footprint metrics to use ctr for kata 2.0

### LINKED ISSUE #3390: update fast footprint metrics to use ctr for kata 2.0

### PR DESCRIPTION
**Pull request by GabyCT**
This PR updates the fast footprint density metrics to use ctr
instead of docker in order to use it for kata 2.0

Fixes #3390

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Review by jcvenegas — APPROVED** [review]

File: metrics/density/fast_footprint.sh [hunk 1]
```diff
@@ -46,25 +37,9 @@ PARALLELISM="${PARALLELISM:-10}"
...
-# of handles in the kernel. Disable log-bin as it hits a file resize fail error.
-#PAYLOAD_ARGS="${PAYLOAD_ARGS:- --innodb_use_native_aio=0 --disable-log-bin}"
-#PAYLOAD_RUNTIME_ARGS="${PAYLOAD_RUNTIME_ARGS:- -m 4G -e MYSQL_ALLOW_EMPTY_PASSWORD=1}"
-#
-#  elasticsearch is a large workload
-#PAYLOAD="${PAYLOAD:-elasticsearch}"
-#PAYLOAD_ARGS="${PAYLOAD_ARGS:-}"
-#PAYLOAD_RUNTIME_ARGS="${PAYLOAD_RUNTIME_ARGS:- -m 8G}"
-#########################
+PAYLOAD_RUNTIME_ARGS="${PAYLOAD_RUNTIME_ARGS:-5120}"
```
Comment (chavafg) [inline metrics/density/fast_footprint.sh:42]: should this be `PAYLOAD_RUNTIME_ARGS="${PAYLOAD_RUNTIME_ARGS:- --memory-limit 5120}`?
BTW, why not 2G as before?

File: metrics/density/fast_footprint.sh [hunk 2]
```diff
@@ -420,7 +322,8 @@ launch_containers() {
 	done
 
 	for n in $(seq 1 $leftovers); do
-		docker run --rm -d --runtime=$RUNTIME $PAYLOAD_RUNTIME_ARGS $PAYLOAD $PAYLOAD_ARGS &
+		containers+=($(random_name))
+		sudo ctr run --memory-limit $PAYLOAD_RUNTIME_ARGS -d --runtime=$CONTAINERD_RUNTIME $PAYLOAD ${containers[-1]} sh -c $PAYLOAD_ARGS &
```
Comment (chavafg) [inline metrics/density/fast_footprint.sh:326]: move `--memory-limit` to the `PAYLOAD_RUNTIME_ARGS` value?

File: metrics/density/fast_footprint.sh (hunk 1 above)
Comment (GabyCT) [reply metrics/density/fast_footprint.sh:42]: @chavafg it is not possible to launch containers with ctr with the amount of memory, it takes a lot of time to launch a container which is something that we do not want for this test

File: metrics/density/fast_footprint.sh (hunk 2 above)
Comment (GabyCT) [reply metrics/density/fast_footprint.sh]: @chavafg done

**Review by chavafg — APPROVED** [review]

**Pull request merged**